life_expectancy/tests/conftest.py

```diff
@@ -20,4 +20,20 @@ def run_before_and_after_tests() -> None:
 @pytest.fixture(scope="session")
 def pt_life_expectancy_expected() -> pd.DataFrame:
     """Fixture to load the expected output of the cleaning script"""
-    return pd.read_csv(FIXTURES_DIR / "pt_life_expectancy_expected.csv")
+    return pd.read_csv(FIXTURES_DIR / "pt_life_expectancy_expected_test.csv")
+
+
+@pytest.fixture(scope="session")
+def loaded_data_test() -> pd.DataFrame:
```

Comment:
Meaningful names: a conftest fixture is already implied to be only for use for testing, so the "_test" suffix is not necessary